run_dir/static/js/qpcr_pools.js

```diff
@@ -57,8 +57,22 @@ function load_table() {
           }));
           tbl_row.append($('<td>').html(function(){
             var to_return = '';
-            $.each( pools['runmodes'], function(i, runmode){
-              to_return = to_return + '<div class="mult-pools-margin">'+ runmode +'</div>'
+            $.each( pools['sequencing_platforms'], function(i, seq_platform){
+              to_return = to_return + '<div class="mult-pools-margin">'+ seq_platform +'</div>'
+            });
+            return to_return;
+          }));
+          tbl_row.append($('<td>').html(function(){
+            var to_return = '';
+            $.each( pools['flowcells'], function(i, flowcell){
+              to_return = to_return + '<div class="mult-pools-margin">'+ flowcell +'</div>'
+            });
+            return to_return;
+          }));
+          tbl_row.append($('<td>').html(function(){
+            var to_return = '';
+            $.each( pools['proj_queue_dates'], function(i, queued_date){
+              to_return = to_return + '<div class="mult-pools-margin">'+ queued_date +'</div>'
```

Comment:
Maybe you could just change the text color (since it's white when you copy it to clipboard)? 

To line 82:

`tbl_row.append($('<td>').html('<span class="text-'+daysAndLabel[1]+'">'+(avg_wait_calc).toFixed(1)+'</span>'));`

Reply:
Changed to alert from badge and the text is not white anymore.